conjure_oxide/src/main.rs

```diff
@@ -125,13 +127,33 @@ fn setup_logging(global_args: &GlobalArgs) -> anyhow::Result<()> {
     Ok(())
 }
 
+fn run_completion_command(completion_args: cli::CompletionArgs) -> anyhow::Result<()> {
+    let mut cmd = Cli::command();
+    let shell = completion_args.shell;
+    let name = cmd.get_name().to_string();
+
+    let shell = match shell {
```

Comment:
Wondering if we could use the existing `Shell` type directly instead of defining our own? Seems identical: https://docs.rs/clap_complete/latest/clap_complete/aot/enum.Shell.html